code/game/objects/effects/mines.dm

```diff
@@ -27,7 +27,7 @@
 	s.start()
 	mineEffect(victim)
 	triggered = 1
-	qdel(src)
+	destroy_effect()

```

Comment:
absolutely all of these need sanity checks so that you can't walk over a mine multiple times. Maybe make them not fade out?